build.gradle

```diff
@@ -1,19 +1,21 @@
 // Temporary workaround, see https://github.com/michel-kraemer/gradle-download-task/issues/56
 buildscript {
-    repositories {
-        jcenter()
-    }
-    dependencies {
-        classpath 'de.undercouch:gradle-download-task:3.1.1'
-        classpath 'org.apache.httpcomponents:httpclient:4.5.2'
-    }
+  repositories {
+    jcenter()
```

Comment:
Whitespace changes are fine to make, but please don't make them in a pull request (just commit them directly to the master branch).  Could you do so, so that this does not show up in the code review?